src/main/knn/KnnGraphTester.java

```diff
@@ -136,6 +137,8 @@ private KnnGraphTester() {
     selectivity = 1f;
     prefilter = false;
     randomCommits = false;
+    quantizeBits = 7;
```

Comment:
I'm surprised the default is 7 and not 8?

Reply:
7 is signed, we do signed quantization which 7 bit

Reply:
8 bit quantization would be unsigned and require unsigned byte comparisons, which we don't have in Lucene yet, we can easily add that in the future, its just frustrating to have optimized SIMD operations for every single option out there.

Reply:
OK, I think I understand. signed quantities do occupy 8 bits, but we call it 7 bits to distinguish from unsigned? I'm not advocating for unsigned here, just trying to understand the terminology